Integration/FRAIHMWORK Integrator Starter Kit Collection.json

```diff
@@ -0,0 +1,2038 @@
+							"event": [
+								{
+									"listen": "test",
+									"script": {
+										"exec": [
+											"pm.test(\"Successful GET with UUID\", function () {\r",
+											"    pm.response.to.have.status(200);\r",
+											"});"
+										],
+										"type": "text/javascript"
+									}
+								},
+								{
+									"listen": "prerequest",
+									"script": {
+										"exec": [
+											""
+										],
+										"type": "text/javascript"
+									}
+								}
+							],
+							"request": {
+								"auth": {
+									"type": "bearer",
+									"bearer": [
+										{
+											"key": "token",
+											"value": "{{access_token}}",
+											"type": "string"
+										}
+									]
+								},
+								"method": "GET",
+								"header": [],
+								"url": {
+									"raw": "{{baseUrl}}{{Mitigation}}/mitigation/",
+									"host": [
+										"{{baseUrl}}{{Mitigation}}"
+									],
+									"path": [
+										"mitigation",
+										""
+									]
+								},
+								"description": "This endpoint allows requesting services to find information about registered instances that FRAIHMWORK is currently aware of"
+							},
+							"response": []
+						}
+					]
+				},
+				{
+					"name": "/{uuid}",
+					"item": [
+						{
+							"name": "GET",
+							"item": [
+								{
+									"name": "Retrieve posted mitigation",
+									"event": [
+										{
+											"listen": "test",
+											"script": {
+												"exec": [
+													""
+												],
+												"type": "text/javascript"
+											}
+										},
+										{
+											"listen": "prerequest",
+											"script": {
+												"exec": [
+													""
+												],
+												"type": "text/javascript"
+											}
+										}
+									],
+									"request": {
+										"auth": {
+											"type": "bearer",
+											"bearer": [
+												{
+													"key": "token",
+													"value": "{{access_token}}",
+													"type": "string"
+												}
+											]
+										},
+										"method": "GET",
+										"header": [],
+										"url": {
+											"raw": "{{baseUrl}}{{Mitigation}}/mitigation/:uuid",
+											"host": [
+												"{{baseUrl}}{{Mitigation}}"
+											],
+											"path": [
+												"mitigation",
+												":uuid"
+											],
+											"variable": [
+												{
+													"key": "uuid",
+													"value": "{{UUIDMitigationResponse}}",
+													"description": "(Required) Uuid of the mitigation posted"
+												}
+											]
+										},
+										"description": "This endpoint allows requesting services to find information about registered instances that FRAIHMWORK is currently aware of"
+									},
+									"response": []
+								}
+							]
+						},
+						{
+							"name": "PUT",
+							"item": [
+								{
+									"name": "Update mitigation to COMPLETED",
+									"event": [
```

Comment:
Changing mitigation state from RECOMMENDED to COMPLETE might not be possible. (Results in a bad request).
Update to ACCEPTED and then COMPLETE. 

Also, update the name of the test to "Update mitigation to COMPLETE".

Reply:
I've reordered the two mitigation examples so ACCEPTED is prior to COMPLETE.